.github/pull_request_template.md

```diff
@@ -20,4 +20,5 @@ Please describe the tests that you ran to verify your changes. Provide instructi
 - [ ] I have performed a self-review of my code
 - [ ] I have commented my code, particularly in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
+- [ ] I have added meaningful description with JIRA/GitHub issue key 
```

Comment:
could you add an info about where exactly should this description be added?

Reply:
Hi @psturc, is it ok for you now?